doc/dev-notes.md

```diff
@@ -1,3 +1,7 @@
+# What version of node is recommended?
+
+To run this in development environment, node version 18 is recommended. 
```

Comment:
Could be useful for devs. But what do you think? What would be the best place to put this information?

Reply:
Specify version 20 and leave it in dev-notes

Reply:
Good idea, specified the version 20